conda/dev.yml

```diff
@@ -0,0 +1,9 @@
+name: zppy_dev
```

Comment:
@tomvothecoder Do `dev.yml` and `meta.yaml` look good to you? I modeled them off their respective files for `zstash` (https://github.com/E3SM-Project/zstash/tree/master/conda). Note that we haven't addressed #10 (CI/CD workflow) yet, so I took out the developer tools.

Reply:
@forsyth2 The template looks good, just need to add dependencies for both files as mentioned in the other comments.